packages/sources/coinpaprika-test/src/endpoint/vwap.ts

```diff
@@ -0,0 +1,114 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { customSettings, DEFAULT_API_ENDPOINT, PRO_API_ENDPOINT } from '../config'
+
+export const inputParameters: InputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
+    type: 'string',
+    required: true,
+  },
+  hours: {
+    description: 'Number of hours to get VWAP for',
+    type: 'number',
+    default: 24,
+  },
+  coinid: {
+    description: 'The coin ID (optional to use in place of `base`)',
+    required: false,
+    type: 'string',
+  },
+}
+
+interface Response {
+  timestamp: string
+  price: number
+  volume_24h: number
+  market_cap: number
+}
+
+export interface RequestParams {
+  coinid?: string
+  base?: string
```

Comment:
Same question here as above, since base is a required input param for this endpoint too.